app/helpers/sellers_helper.rb

```diff
@@ -3,7 +3,9 @@
 module SellersHelper
   def self.generate_seller_json(seller:)
     locations = seller.locations
+    recipient = seller.recipient
     seller = seller.as_json
+    seller['recipient'] = recipient.as_json if recipient != nil
```

Comment:
nit unless recipient.nil?

Reply:
Can we call it can_buy_a_meal_for? Right now it's not really clear why a Contact is related to a seller

Reply:
or maybe partner? distributor? buy_a_meal_for_partner? buy_a_meal_for_distributor?